src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and move your app to production, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When users edit their Koji profile, they can choose to add an app from the App Store.
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
+
+== Creating an App Store listing
+
+You need the following for your App Store listing:
+
+* A 500x500-pixel icon to be displayed with your listing in the App Store.
+* A 1200x630-pixel thumbnail to be displayed when your listing is shared in other social media sites.
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
```

Comment:
Spacing problem with this include. Step 2, "Click My App Templates." is showing on the same line. You probably need something like this:
```suggestion
. {blank}
include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
```

Reply:
Step 1 is supposed to have two sentences (two combined reusables). The period in the middle is a side-effect of the external-link icon, which the build automatically inserts. Maybe the build inserts an extra space after that icon.

Reply:
Perhaps your reusables are too small. The reusables works best as chunks. Let's take a look together.